business/input.go

```diff
@@ -0,0 +1,72 @@
+package business
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+var (
+	HFlag      = flag.Bool("h", false, "Show help")
+	WordSubCmd = flag.NewFlagSet("word", flag.ExitOnError)
+	EFlag      = WordSubCmd.Bool("e", false, "Display the word in a sentence")
+	TFlag      = WordSubCmd.Bool("t", false, "Display only thesaurus results")
+	UDFlag     = WordSubCmd.Bool("ud", false, "Also query Urban Dictionary for results")
+
+	LookupValue string
+)
+
+func manual() {
+	fmt.Fprintf(flag.CommandLine.Output(), "\n")
+	fmt.Fprintf(flag.CommandLine.Output(), "%s\n", "Dicthesaurus requires only a single command: the word you want to search for.")
+	fmt.Fprintf(flag.CommandLine.Output(), "\n")
+	fmt.Fprintf(flag.CommandLine.Output(), "%s\n", "\"dt linux\"")
+	fmt.Fprintf(flag.CommandLine.Output(), "\n")
+	fmt.Fprintf(flag.CommandLine.Output(), "%s\n", "FLAGS")
+	fmt.Fprintf(flag.CommandLine.Output(), "\t%s\n", "-e   :  Include the word used in a sentence")
+	fmt.Fprintf(flag.CommandLine.Output(), "\t%s\n", "-t   :  Only return a word's result from the thesaurus")
+	fmt.Fprintf(flag.CommandLine.Output(), "\t%s\n", "-ud  :  Return defintions from Urban Dictionary")
+	fmt.Fprintf(flag.CommandLine.Output(), "\n")
+}
+
+func subcommand() string {
+	return os.Args[1]
+}
+
+func usage() {
```

Comment:
"usage" makes sense if you understand the error specific function of the "flag" standard package, but it could be better defined:

"parsingError"
"parsingErrorInstructions"
"parsingErrHelp"
"parsingErrorHelp"